run_capture.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+import time
+import argparse
+import cv2
+import os
+from capture_image import CaptureImage
+from video_live_generator import VideoLiveGenerator
+from power_port_detector import PowerPortDetector
```

Comment:
Detector should not be needed. The corresponding detector files should be removed as well.